rpc/wrtc_client_channel_test.go

```diff
@@ -538,3 +538,53 @@ func TestWebRTCClientChannelWithInterceptor(t *testing.T) {
 
 	wg.Wait()
 }
+
+func TestWebRTCClientChannelCanStopStreamRecvMsg(t *testing.T) {
+	// Regression test for RSDK-4473.
+	testutils.SkipUnlessInternet(t)
+	logger := golog.NewTestLogger(t)
+	pc1, pc2, dc1, dc2 := setupWebRTCPeers(t)
+
+	clientCh := newWebRTCClientChannel(pc1, dc1, logger, nil, nil)
+	defer func() {
+		test.That(t, clientCh.Close(), test.ShouldBeNil)
+	}()
+	serverCh := newBaseChannel(context.Background(), pc2, dc2, nil, logger)
```

Comment:
We use base channels to represents server channels in this test file. We _could_ make a full server channel, but it requires [some extra logic](https://github.com/viamrobotics/goutils/blob/main/rpc/wrtc_server_channel_test.go#L31-L43) setting up the actual server, that I felt was unnecessary, as what we're testing here is really client-side.